firebase-firestore/src/main/java/com/google/firebase/firestore/remote/GrpcCallProvider.java

```diff
@@ -50,6 +52,10 @@
 
   private CallOptions callOptions;
 
+  // A timer that elapses after CONNECTIVTY_ATTEMPT_TIMEOUT_MS, at which point we close the
```

Comment:
Same comments as on CONNECTIVITY_ATTEMPT_TIMEOUT_MS; we don't seem to be closing the stream.

I suggest investing in a good description of exactly how this works in one location and then referring to that with something like "See $location for more details about how this works."

Reply:
Removed private method comments and consolidated the description in `onConnectivityStateChange`